moonstone/filtering/basics_filtering.py

```diff
@@ -59,6 +59,17 @@ def _exclude_names(self):
         return self.df.drop(self.names, axis=self.axis)
 
     def filter(self) -> pd.DataFrame:
+        # intersection of names to remove or keep and index's/columns' names
+        tmp = len(self.names)
+        if self.axis == 0:
+            self.names = list(self.df.index.intersection(self.names))
+        else:
+            self.names = list(self.df.columns.intersection(self.names))
+        if tmp - len(self.names) == 1:
```

Comment:
I think only the second test (in the elif) is enough but instead of comparing to 1 you can compare to 0 

If only one is not found, you will have a list of one item which is ok for log message.

Reply:
It was just because it was weird for me if the log message wrote "1 names were not found in the dataframe"

Reply:
oh true, I see the problem sorry. you can use ("name(s)") maybe?